Task2/js/app.js

```diff
@@ -1,37 +1,22 @@
 document.addEventListener('DOMContentLoaded', () => {
-    ZOMBIE_TYPE.DEAD = 0;
-
     const zombieList = document.querySelector('.zombies');
 
-    renderZombieList(zombies);
-
-    function renderZombieList(units) {
-        zombieList.innerHTML = '';
-
-        for (let i = 0; i < units.length; i++) {
-            renderZombie(units[i], i);
-        }
-    }
-
-    function renderZombie(item, index) {
+    for (let i = 0; i < zombies.length; i++) {
         const zombie = document.createElement('div');
 
         zombie.classList.add('zombie');
 
-        if (item.type === ZOMBIE_TYPE.SMALL) {
+        if (zombies[i].type === ZOMBIE_TYPE.SMALL) {
             zombie.classList.add('zombie--small');
-        } else if (item.type === ZOMBIE_TYPE.MAD) {
+        } else if (zombies[i].type === ZOMBIE_TYPE.MAD) {
             zombie.classList.add('zombie--mad');
-        } else if (item.type === ZOMBIE_TYPE.STRONG) {
+        } else if (zombies[i].type === ZOMBIE_TYPE.STRONG) {
             zombie.classList.add('zombie--strong');
-        } else if (item.type === ZOMBIE_TYPE.DEAD) {
-            zombie.classList.add('zombie--dead');
         }
 
         zombie.addEventListener('click', function (event) {

```

Comment:
please implement a solution without using any parameter of this function

Reply:
done